sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_summary_v1/query.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+  submission_date,
+  service,
+  country,
+  LANGUAGE,
+  app_version,
+  os_name,
```

Comment:
Are major and minor versions required? If yes, you could use https://github.com/mozilla/bigquery-etl/blob/main/sql/mozfun/norm/truncate_version/udf.sql

Reply:
thanks, actually we probably don't care about specific patch versions 🤔 I added truncation to minor version.